closed/OpenJ9.gmk

```diff
@@ -208,14 +208,15 @@ $(call openj9_add_jdk_shlibs, java.base, \
 	j9vmchk29 \
 	j9vrb29 \
 	j9zlib29 \
-	jclse11_29 \
+	jclse12_29 \
 	omrsig \
 	)
 
 ifeq (windows,$(OPENJDK_TARGET_OS))
-$(eval $(call openj9_add_jdk_rules, \
-	$(OUTPUTDIR)/vm/lib/$(call STATIC_LIBRARY,jvm), \
-	$(OUTPUTDIR)/vm/j9vm_jdk12/$(call STATIC_LIBRARY,jdk12_jvm)))
+$(foreach subdir, j9vm server, \
+	$(eval $(call openj9_add_jdk_rules, \
+		$(JDK_OUTPUTDIR)/bin/$(subdir)/$(call STATIC_LIBRARY,jvm), \
+		$(OUTPUTDIR)/vm/j9vm_jdk12/$(call STATIC_LIBRARY,jdk12_jvm))))
```

Comment:
Should this be jvm instead of jdk12_jvm?

Reply:
No, the redirectors are all the same after https://github.com/eclipse/openj9/pull/3638, but the main JVM libraries are still version-specific.